charts/umbrella/README.md

```diff
@@ -471,9 +471,19 @@ helm install -f values-adopter-portal.yaml umbrella . --namespace umbrella --cre
 
 Involved components:
 
-EDC, MIW, DTR, Vault (data provider and consumer in tx-data-provider), CentralIdP.
+CentralIdP, DBRS-Server-Memory, Data Consumer, Data Provider, SSI Credential Issuer, SSI Dim Wallet Stub.
 
-TBD.
+To start the Data Exchange, it is necessary to execute the involved components:
```

Comment:
Please reference to this part of the readme https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/umbrella-2.0.0/charts/umbrella/README.md#option-2 or https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/umbrella-2.0.0/charts/umbrella/README.md#option-2-1 respectively to not maintain helm install commands in multiple places